examples/dt/osasinfra/control-plane.md

```diff
@@ -3,6 +3,39 @@
 ## Assumptions
 
 - A storage class called `local-storage` should already exist.
+- Cluster observability operator is already deployed. If not, follow the
+  steps found [below](#cluster-observability-operator).
+
+### Cluster observability operator
+
+Cluster Observability Operator must be installed as it is required by OpenStack
+Telemetry operator. If not installed, the below steps can be followed
+
+```bash
+cat > subscription.yaml << EOF
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: observability-operator
+  namespace: openshift-operators
+  labels:
+    operators.coreos.com/observability-operator.openshift-operators: ""
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+# Apply the cr
+oc apply -f subscription.yaml
+
+# Wait for the deployment to be ready
+oc wait deployments/observability-operator --for condition=Available \
+    --timeout=300s
+```
```

Comment:
This is also being addressed in another PR [1] and I think we'll want to merge that one first.  Then we won't need to have this documented step here.

[1] https://github.com/openstack-k8s-operators/architecture/pull/361#discussion_r1803685787

Reply:
https://github.com/openstack-k8s-operators/architecture/pull/361 has since merged

Reply:
@mandre Please rebase and then you can remove this docs section, as COO is now installed automatically

Reply:
I think we're still waiting this patch to be updated now that 361 has merged.

Reply:
Done. Sorry for the lag, I was away last week :palm_tree: